mix.exs

```diff
@@ -67,7 +67,7 @@ defmodule Cog.Mixfile do
      {:hedwig, "~> 0.3.0"},
      {:gproc, "~> 0.5.0", override: true},
      {:html_entities, "~> 0.3.0"},
-     {:spanner, github: "operable/spanner", ref: "106b9371fa77c9dcfbdb758598cb3beda13c30f2"},
+     {:spanner, github: "operable/spanner", ref: "c5b4e457f3a91434752b091b101c71eaf5d51940"},
```

Comment:
Might as well switch over to using `master` here, too, if we're going to do that elsewhere.